packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -498,6 +510,36 @@ describe("SharedTree", () => {
 		});
 	});
 
+	it("can load from summary", async () => {
```

Comment:
This is a simplified version of the following tests, which tests the simplest case of loading (no concurrent/pending ops) useful for root causing summary loading issues: we already had test coverage for this via the more complex cases, but having a simpler tests as well is nice.